buildSrc/src/main/groovy/net/danlew/android/joda/TzDataPlugin.groovy

```diff
@@ -51,7 +51,7 @@ class TzDataPlugin implements Plugin<Project> {
                 @Override
                 boolean accept(File pathname) {
                     // Dumb check: if it has a dot, ignore it
-                    return !pathname.absolutePath.contains(".")
+                    return !pathname.name.contains(".")
```

Comment:
Took me longer than I care to admit to find this - my OS username contains a dot, so all tzdata files were rejected. Switched to only checking filename instead of full path.